FavoritePage

D3mk4-L1f3 opened the pull request:
Create card + list components

D3mk4-L1f3 committed:
FavoritePage

Create card + list components

Redddeve commented on src/images/trash.svg:
Лишня іконка? Чому не використати icon-trash з sprite.svg?

Redddeve commented on src/components/CocktailItem/CocktailItem.styled.js:
```diff
@@ -0,0 +1,77 @@
+  color: rgba(243, 243, 243, 0.5);
+`;
+
+export const CocktailDescriptionContainer = styled.div`
```
Навіщо цей дів? Без нього виглядає точно так само.

Redddeve commented on src/components/CocktailItem/CocktailItem.styled.js:
```diff
@@ -0,0 +1,77 @@
+`;
+
+export const CocktailLabel = styled.p`
+  color: rgba(243, 243, 243, 0.5);
```
цей колір є в ::root (--transp-white), краще замінити для кращої читабельності.

Redddeve commented on src/components/CocktailItem/CocktailItem.styled.js:
```diff
@@ -0,0 +1,77 @@
+  height: 75px;
+`;
+
+export const CocktailDescription = styled.p`
```
Коли доробиш сторінку тут можна підправити стилі щоб не було такого як на картинці.
![image_2023-12-15_233620236](https://github.com/Redddeve/project-drink-master/assets/127204854/0ea064cb-5eef-4c4a-a2ab-ec25c9678121)
В кінці повинно бути як на макеті, якщо текст не вміщуюється то додаються три точки

Redddeve commented on src/components/CocktailItem/CocktailItem.jsx:
```diff
@@ -0,0 +1,39 @@
+        <CocktailTitle>{drink}</CocktailTitle>
+        <CocktailLabel>{alcoholic}</CocktailLabel>
+      </CocktailContainerTitle>
+      <CocktailDescriptionContainer>
```
DescriptionContainer нічого не робить

Redddeve reviewed (changes requested):
Хочу почути чому додався ще один svg файл і трохи поправити CocktailItem і його стилі.

Redddeve commented on src/components/CocktailItem/CocktailItem.styled.js:
```diff
@@ -0,0 +1,77 @@
+export const Icon = styled.img`
+  width: 24px;
+  height: 24px;
+  fill: white;
```
Чому тут не svg, а img?

D3mk4-L1f3 committed:
Merge branch 'main' into feature/favoriteDrinksPage

D3mk4-L1f3 committed:
Merge branch 'main' into origin/feature/favoriteDrinksPage

D3mk4-L1f3 committed:
Merge branch 'main' into origin/feature/favoriteDrinksPage

D3mk4-L1f3 committed:
update styles of favorite page

Виправлено всі попередні зауваження + додана адаптивна складова та картинка для пустого списку "улюблене"

D3mk4-L1f3 committed:
Delete cocktail.json

Redddeve commented on src/pages/FavoriteDrinksPage/FavoriteDrinksPage.styled.js:
```diff
@@ -1,8 +1,15 @@
   font-weight: 600;
   line-height: 1.18;
+
+  @media screen and (min-width: 768px) {
```
Можна замінити таким чином:
```suggestion
 @media ${device.tablet} {
```

Redddeve commented on src/images/cocktail.json:
Цей json тут не потрібен. Видалити при комміті

D3mk4-L1f3 committed:
update @media

Redddeve commented on src/components/FavoriteCocktails/FavoriteCocktails.styled.js:
```diff
@@ -0,0 +1,38 @@
+
+  @media ${device.tablet} {
+    font-size: 16px;
+  }
```
Щойно протестив, це не треба. Сайз наслідується з body

Redddeve reviewed (approved):
Good work, couple changes would be welcomed
